kolibri/plugins/coach/assets/src/views/lessons/LessonsRootPage.vue

```diff
@@ -96,10 +96,14 @@
                         :style="{ display: 'inline-block', marginLeft: '6px' }"
                         :size="26"
                       />
+                      <span :id="`toggle-lesson-visibility-label-${lesson.id}`">
+                        Toggle visibility for lesson {{ lesson.name }}
+                      </span>
                       <KSwitch
-                        v-else
-                        :key="`switch-${lesson.id}`"
+                        v-else:
+                        key="`switch-${lesson.id}`"
```

Comment:
I believe this piece of code is causing an error in my code editor. Is there a particular reason that the `:` colon was moved from in front of `key` and positioned after `v-else`?